pentasharp/appsettings.json

```diff
@@ -5,5 +5,10 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "DefaultDbConnection": "Server=(localdb)\\MSSQLLocalDB;Database=PentasharpDB;Trusted_Connection=True;MultipleActiveResultSets=true"
```

Comment:
in Program.cs you are using DefaultConenction
    builder.Services.AddDbContext<AppDbContext>(options =>
                options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection")));
 , here you are referring as DeafultDBConnection. The name should match